test/helpers/helpers.go

```diff
@@ -542,3 +543,40 @@ func CheckSubscriptionNotUpdated(client test.FrameworkClient, name, namespace st
 	}
 	return nil
 }
+
+var isCoAPIPresent *bool
```

Comment:
Shouldn't this be set at the top of this file, maybe as part of a var () block?

Reply:
My mistake - I had meant to move this as you suggested.